packages/blocks/src/components/RadioCard/RadioCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import cx from 'classnames';
+
+type RadioCardProps = {
+  startIcon?: React.ReactNode;
+  isSelected: boolean;
+  label: string;
+};
+
+export const RadioCard: React.FC<RadioCardProps> = ({
+  startIcon,
+  isSelected,
+  label,
+}) => {
+  return (
+    <div className={cx('t-radio-card', {'t-tag-blue': isSelected})}>
+      {startIcon && startIcon}
+      {label && label}
+    </div>
+  );
+};
```

Comment:
For `isSelected = false` can we add an `active` state? A subtle indicator to show it's being pressed?